.github/workflows/dependency-review.yml

```
@@ -26,4 +26,4 @@ jobs:
           license-check: true
           
           # Allowed licenses (common open source licenses)
-          allow-licenses: MIT, Apache-2.0, BSD-2-Clause, BSD-3-Clause, ISC, 0BSD, CC0-1.0, CC-BY-3.0, CC-BY-4.0, Unlicense
\ No newline at end of file
+          allow-licenses: MIT, Apache-2.0, BSD-2-Clause, BSD-3-Clause, ISC, 0BSD, CC0-1.0, CC-BY-3.0, CC-BY-4.0, Unlicense, LicenseRef-scancode-unicode, Unicode-License
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)